sherlock.py

```diff
@@ -90,7 +90,8 @@ def sherlock(username, verbose=False, tor=False, unique_tor=False):
     }
 
     # Load the data
-    with open("data.json", "r", encoding="utf-8") as raw:
+    script_directory = os.path.dirname(os.path.realpath(__file__))
+    with open(script_directory + "/data.json", "r", encoding="utf-8") as raw:
```

Comment:
This will break on Windoze I believe...  Instead of hardcoding the /, would it be possible to __os.path.join()__ like __filename = os.path.join(os.path.dirname(os.path.realpath(__file__)), "data.json")__

Reply:
The change would work on Windows, but I agree with @cclauss that using os.path.join() would be better.